src/content/blog/javax-jakarta-openrewrite-automigrate/index.md

```diff
@@ -0,0 +1,154 @@
+#Auto-migration of apps from Java EE 8 to Jakarta EE 9 using Windup and Openrewrite
```

Comment:
Please add the metadata section at the beginning of the file.
Something like
```suggestion
---
title: Auto-migration of apps from Java EE 8 to Jakarta EE 9 using Windup and Openrewrite
description: Auto-migration of apps from Java EE 8 to Jakarta EE 9 using Windup and Openrewrite
featured: true
author: m-brophy
date: 2022-08-17T00:00:00Z
tags:
  - docs
---
```
with:

- `title` being the title shown in the web page
- `description` is for SEO (so having it like `title` makes sense to me)
- `featured` makes it appear in the `Featured Posts` right menu in blog web page
- `author` is not yet used but we agreed on using the GH username
- `date` is set to the expected/planned release day

Another note we figured out yesterday is that `#` (single) is not allowed in the body of the blog post hence the "bigger" heading allowed is `##`